packages/dev/core/src/node.ts

```diff
@@ -588,10 +596,9 @@ export class Node implements IBehaviorAware<Node> {
             return;
         }
         this._nodeDataStorage._isEnabled = value;
+        this._syncParentEnabledState();

```

Comment:
Why this move?

Reply:
Oh, this is because when some object is enable/disabled, but that hasn't sync up to its children at the time _onEnabledStateChangedObservable notify observers, the accessibility updates (which observes it) won't get the correct tree.

Reply:
Makes sense!